docs/guides/external-mgs/readme.md

```diff
@@ -0,0 +1,151 @@
+
+```yaml
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfStorageProfile
+metadata:
+  name: persistent-lustre-shared-mgs
+  namespace: nnf-system
+data:
+[...]
+  lustreStorage:
+    externalMgs: ""
+    combinedMgtMdt: true
+    standaloneMgtPoolName: ""
+[...]
+```
+
+The persistent storage is created with the following DW directive
+
+```bash
+#DW create_persistent name=shared-lustre capacity=100GiB type=lustre profile=persistent-lustre-shared-mgs
+```
+
+After the persistent Lustre file system is created, an admin can discover the MGS address by looking at the `NnfStorage` resource with the same name as the persistent storage that was created (`shared-lustre` in the above example).
+
+```yaml
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfStorage
+metadata:
+  name: shared-lustre
+  namespace: default
+[...]
+status:
+  mgsNode: 5.6.7.8@eth1
+[...]
+```
+
+A separate `NnfStorageProfile` can be created that specifies the MGS address.
+
+```yaml
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfStorageProfile
+metadata:
+  name: internal-mgs
+  namespace: nnf-system
+data:
+[...]
+  lustreStorage:
+    externalMgs: 5.6.7.8@eth1
+    combinedMgtMdt: false
+    standaloneMgtPoolName: ""
+[...]
+```
+
+With this configuration, an admin must determine that no file systems are using the shared MGS before destroying the persistent Lustre instance.
+
+## Configuration with an Internal MGS Pool
+
+Another method NNF supports is to create a number of persistent Lustre MGTs on NNF nodes. These MGTs are not part of a full file system, but are instead added to a pool of MGTs available for other Lustre file systems to use. Lustre file systems that are created will choose one of the MGSs at random to use and add a reference to make sure it isn't destroyed. This configuration has the advantage of spreading the Lustre management load across multiple servers and keeping MGS traffic within the high speed network. The disadvantage of this configuration is that is does not provide high availability.
+
+To configure the system this way, the first step is to make a pool of Lustre MGTs. This is done by creating a persistent instance from a storage profile that specifies the `standaloneMgtPoolName` option. This option tells NNF software to only create an MGT, and to add it to a named pool. The following `NnfStorageProfile` provides an example where the MGT is added to the `example-pool` pool:
+
+```yaml
+apiVersion: nnf.cray.hpe.com/v1alpha1
+kind: NnfStorageProfile
+metadata:
+  name: mgs-pool-member
+  namespace: nnf-system
+data:
+[...]
+  lustreStorage:
+    externalMgs: ""
+    combinedMgtMdt: false
+    standaloneMgtPoolName: "example-pool"
+[...]
+```
+
+The persistent storage is created with the following DW directive:
+
+```bash
+#DW create_persistent name=mgt-pool-member-1 capacity=1GiB type=lustre profile=mgt-pool-member
```

Comment:
Above, you named the profile "mgs-...", but here you're looking for "mgt-..." ,   so again, you're loosy-goosy with mgt vs mgs.